From 239d7feceb6ea679da37bad2b9039fd39ac3fd7c Mon Sep 17 00:00:00 2001
From: Ubuntu <ubuntu@ip-172-31-27-164.ap-northeast-2.compute.internal>
Date: Tue, 29 Jan 2019 18:54:58 +0900
Subject: [PATCH 1/3] ndk: replaced r13b with r12b

This commit is to synchronize the version of NDK toolkit between android-gstreamer
and fall detection (dvs-parser-server).

**Changes**
1. replaced NDK r13b with r12b using gstreamer 1.12.4.

Signed-off-by: Geunsik Lim <geunsik.lim@samsung.com>
---
 cerbero/bootstrap/android.py | 2 +-
 config/android.config        | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/cerbero/bootstrap/android.py b/cerbero/bootstrap/android.py
index d875309..c3cf1a5 100644
--- a/cerbero/bootstrap/android.py
+++ b/cerbero/bootstrap/android.py
@@ -27,7 +27,7 @@ from cerbero.utils import _, shell
 class AndroidBootstrapper (BootstrapperBase):
 
     NDK_BASE_URL = 'http://dl.google.com/android/repository/'
-    NDK_VERSION = 'r13b'
+    NDK_VERSION = 'r12b'
     NDK_ZIP = 'android-ndk-' + NDK_VERSION + '-%s-%s.zip'
 
     def start(self):
diff --git a/config/android.config b/config/android.config
index 6b6110d..22c3e65 100644
--- a/config/android.config
+++ b/config/android.config
@@ -14,7 +14,7 @@ variants += ['nopython', 'notestspackage']
 allow_system_libs=False
 
 if not toolchain_prefix:
-    toolchain_prefix = os.path.join(home_dir, 'android-ndk-r13b')
+    toolchain_prefix = os.path.join(home_dir, 'android-ndk-r12b')
 
 toolchain_path = None
 toolchain_version = None
-- 
2.7.4

